doc/api/nmo-query.md

```diff
@@ -0,0 +1,27 @@
+nmo-query(3) -- Create Mango indexes or query existing ones
+====================================================
+
+## SYNOPSIS
+
+    nmo.commands.query([<clusterurl> || <cluster>], <database>, selector)
+    nmo.commands.create([<clusterurl> || <cluster>], <database>, [fields])
```

Comment:
it would be:

```
nmo.commands.query.create
nmo.commands.query.query
```

maybe better names are:

```
nmo.commands.query.run
nmo.commands.query.createindex
```